pkg/api/graph/types.go

```diff
@@ -345,7 +345,7 @@ func BuildConfig(g MutableUniqueGraph, config *build.BuildConfig) graph.Node {
 			in := ImageStreamTag(g, defaultNamespace(from.Namespace, config.Namespace), name, tag)
 			g.AddEdge(in, node, BuildInputImageGraphEdgeKind)
 		case "ImageStreamImage":
-			glog.V(4).Infof("Ignoring ImageStreamImage reference in buildconfig %s/%s", config.Namespace, config.Name)
+			glog.V(4).Infof("Ignoring image stream image reference in build configuration %s/%s", config.Namespace, config.Name)
```

Comment:
ImageStreamImage is object name in origin, I'd leave it as is, buildconfig should be uppercased and also be the name of the object.